src/mots/cli.py

```diff
@@ -156,6 +157,52 @@ def export(args: argparse.Namespace) -> None:
             f.write(output)
 
 
+def set_default(args: argparse.Namespace):
+    """Set a specified settings variable to the provided value."""
+    key = args.key[0]
+    value = args.value[0]
+    with settings.OVERRIDES_FILE.open("r") as f:
+        overrides = yaml.load(f) or {}
+
+    if key not in settings.DEFAULTS:
+        raise ValueError(f"{key} does not exist in defaults.")
+
+    _type = type(settings.DEFAULTS[key])
+
+    if _type(value) == overrides.get(key):
+        raise ValueError(f"{key} is already set to {value} in overrides file.")
+
+    overrides[key] = _type(value)
+    print(f"{key} is now set to {value} ({_type.__name__}) in overrides file.")
+
+    with settings.OVERRIDES_FILE.open("w") as f:
+        yaml.dump(overrides, f)
+
+
+def get_default(args: argparse.Namespace):
+    """Print the value of a specified settings variable.
+
+    If no key is provided, prints all available settings variables.
+    """
+    if not args.key:
+        for key, value in settings.settings.items():
+            print(f"{key} -> {value} ({type(value).__name__})")
+        return
+
+    key = args.key[0]
+
+    with settings.OVERRIDES_FILE.open("r") as f:
+        overrides = yaml.load(f) or {}
+
+    if key not in settings.DEFAULTS:
+        raise ValueError(f"{key} does not exist in defaults.")
+
+    if key in overrides:
+        print(f"{key} is set to {overrides[key]} in settings file.")
```

Comment:
Maybe we could just print the value here? It's not outlined why these CLI commands are needed in the bug or the commit message, but if I'm correct in assuming we want them for some kind of programmatic access to the config, it would be easier to just return the value of the config for parsing. Maybe we could add more details behind a `--verbose` flag.

Reply:
The `get-defaults` command is for convenience, so folks can see their settings easily (both what's in the overrides and the defaults). Everything in the `cli` module is meant to be related to cli interaction and not necessarily for programmatic access (that can be done via Python bindings). However, it could be a plus to have this in a format that is easily consumed if needed. I'll think about this.

Reply:
Updated the bug for clarity as well.